submissions/Oleksii_Drohachov/DOM_task/script.js

```diff
@@ -0,0 +1,175 @@
+      '&emsp;Slaanesh, also known as the "Dark Prince," the "Prince of Pleasure," the "Lord of Excess," the "Perfect Prince," and "Prince of Chaos" in the Imperium of Man and "She Who Thirsts" among the Aeldari, is the Chaos God of pleasure, hedonism, excess and decadence. Lust, pride and self-indulgence are the hallmarks of all who follow it. <br><br>&emsp; Slaanesh is the Lord of Pleasure, the Dark God dedicated to the pursuit of earthly gratification and the overthrow of all decent behaviour, as well as hedonism and pleasure for its own sake. It is the god of obsession, the Master of Excess in All Things, from gluttony to lust to megalomania. Its sacred number is six and the colours associated with Slaanesh are riotous purples, pinks and black. The daemonic armies of Slaanesh are known as the Legions of Excess.',
+    mainLegion: "Main legion: Emperors children",
+  },
+];
+
+function makeFirstPage() {
+  const container = document.createElement("div");
+  container.classList.add("first__page");
+
+  const firstImg = document.createElement("img");
+  firstImg.setAttribute("srcset", greetings.imgUrl);
+  firstImg.classList.add("content__image");
+
+  const text = document.createElement("p");
+  text.classList.add("description");
+  text.innerHTML = greetings.text;
+
+  container.append(firstImg);
+  container.append(text);
+  contentBox.innerHTML = "";
+  contentBox.append(container);
+}
+
+makeFirstPage();
+
+function makeMenu(content) {
+  content.forEach((item) => {
+    const menuItem = document.createElement("li");
+    const menuLink = document.createElement("a");
+    menuItem.classList.add("menu__item");
+    menuLink.setAttribute("href", "#");
+    menuLink.classList.add("menu__link");
+    menuLink.textContent = item.name;
+    menuItem.append(menuLink);
+    menuList.append(menuItem);
+  });
+
+  menuWrapper.append(menuList);
+}
+
+makeMenu(content);
+
+burger.addEventListener("click", menuHandler);
+const menuClasses = menuList.classList;
+
+function menuHandler() {
+  function openMenu() {
+    menuClasses.remove("menu__list_hidden");
+  }
+
+  function closeMenu() {
+    menuClasses.add("menu__list_hidden");
+  }
+
+  function closeMenuByTap(e) {
+    if (!e.target.classList.contains("menu__item")) {
+      menuClasses.add("menu__list_hidden");
+    }
+  }
+
+  if (menuClasses.contains("menu__list_hidden")) {
+    openMenu();
+    contentBox.addEventListener("click", closeMenuByTap);
+  } else {
+    closeMenu();
+    contentBox.removeEventListener("click", closeMenuByTap);
+  }
+}
+
+menuList.addEventListener("click", prepareLayoutById);
+
+function prepareLayoutById(e) {
+  menuHandler();
+
+  const contentId = getContentId();
+
+  prepareLayout(contentId);
+
+  function getContentId() {
+    let result;
+    content.forEach((item) => {
+      if (item.name === e.target.textContent) result = content.indexOf(item);
+    });
+    return result;
+  }
+
+  function prepareLayout(contentId) {
+    if (contentId != undefined) {
```

Comment:
Look, you have an indentation for the whole block of code just because of this one `if`. You can redo it as a single-line `if` like this:

```js
if (!contentId) return
```

And just continue your code on the same level of indentation.